package.json

```diff
@@ -1,9 +1,9 @@
 {
   "name": "codemirror-contrib",
-  "version": "0.0.0",
+  "version": "0.0.1",
   "description": "CodeMirror community contributions",
   "scripts": {
-    "build": "rollup --format umd -n CodeMirror src/**/*.js -o lib",
+    "build": "rollup --format umd -n CodeMirror src/mode/meta.js -o lib/contrib.js",
```

Comment:
yes, me too

wanted to get the abap mode working, then next step we can look into the `rollup` setup?